src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -1708,71 +1708,128 @@ Below is an example for adding a certificate to a Java cacerts file:
+
+More detailed information on how to create or modify Workflows and Buckets are available in the link:../admin/ProActiveAdminGuide.html#_catalog[Admin guide].
+
+=== Catalog security and groups management
+
+The Catalog is secured with a sessionId. Every request to the Catalog must have a valid sessionId inside the request header.
+The security mechanism is based on groups, the group GROUP:public-objects can be access by any user with a valid sessionId. Other groups can be accessed (read and write) if the sessionId corresponds to a user which is part of that group.
+As an example, a user which is part of the interns group (GROUP:interns) can see and change buckets and their contents which belong to the GROUP:interns.
+A user which gets a list of buckets can only see the buckets to which the user belongs plus the GROUP:public-objects associated buckets.
+
+The sessionId is validated against the Scheduler REST api, so if the Catalog does not have access (network outage, firewall) to the Scheduler, no access to the Catalog is possible.
+
+=== Retrieving and searching the workflows
+
+==== Get workflow from REST API
+
+From REST API it's possible to retrieve the raw content of a specific revision of workflow. For this case it's required to specify `name` and `commit_time` of the workflow's  revision:
+
+    GET http://localhost:8080/catalog/buckets/{bucketId}/resources/{name}/revisions/{commit_time}/raw
+
+In order to get the raw content of a last revision of a workflow use next REST API endpoint:
+
+    GET http://localhost:8080/catalog//buckets/{bucketId}/resources/{name}/raw
+
+In this case you need to provide only the `name` of a workflow.
+
+NOTE: A complete documentation of the Catalog REST API is available by default on:
+  http://localhost:8080/catalog/swagger-ui.html
+
+==== GraphQl usage
+
+http://graphql.org/[GraphQL] is the standardized query language, which  provides opportunity to search and retrieve the workflow by specified criterion from the Catalog.
+NOTE: Please follow the graphiql endpoint: `http://localhost:8080/catalog/graphiql` in order to query for the specific workflow.
+
+== Usage of catalog from GUI
+
+The GUI interaction with the Catalog can be done in two places: Studio portal and Workflow Catalog Portal.
+The portals follow the concepts of catalog: workflows store inside buckets, the workflow can have several revisions, each workflow have specific description.
+The view of workflows from Catalog in Studio and from Workflow Catalog Portal follow the same view logic. The page is separated in 3 sections.
+The approach is selecting the needed information from left to right. First user should choose the required bucket, then the user can observe all the workflows in the selected bucket. When the user select the workflow it will be shown all existing revisions of the workflow. And finally there is a description about workflow revision.
```

Comment:
And finally there is a description about workflow revision.
=> By selecting a revision you will be able to see the the description about the workflow revision.